### DISCUSSION TITLE
Add "Restore Closed Tab" as a Keyboard Shortcut

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#795] [Ideas]
### What feature would you like to see?

Essentially, if a tab is (by mistake) closed, being able to reopen the last closed tab via a keyboard shortcut. (eg. ctrl+z)

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/574_

**Comment by thelazyoxymoron** [comment]
On Mac, `Ctrl + Shift + t` is working, looks like `Ctrl` is hardcoded there as a shortcut. `Command + Shift + t` used to work in earlier versions but stopped working in version `1.0.1-a.3`.

**Comment by fenevadkan** [comment]
I fully agree, it would be really important. Maybe the only things that is really missing compared to Firefox.
Please add ctrl-shift-T to reopen closed tabs in recent orders if pressed multiple times, same as in Firefox
